File: Assets/Scripts/Damage.js
var player : GameObject;
var steam : Transform;

function OnTriggerEnter (player : Collider) {
	if(player.tag == "Player") {
		Debug.Log("ANAL");
		if(SteamSystem.currentSteams==0) {
			Debug.Log("DEAD");
			SteamSystem.currentSteams--;
		}
		else {
			Losing_Steams.bulletsOnMoment = SteamSystem.currentSteams;
			Losing_Steams.damaged = true;
			Debug.Log("ANAL");
			player.gameObject.GetComponent(SteamSystem).currentSteams = 0;
			
		}
		Destroy(gameObject);
	}
}

File: Assets/MainScene/Scripts/Damage.cs
using UnityEngine;
using System.Collections;

public class Damage : MonoBehaviour {
	
	public GameObject player;
	public Transform steam;

	void OnTriggerEnter (Collider player) {
		if(player.tag == "Player") {
			if(SteamSystem.currentSteams==0) {
				Debug.Log("DEAD");
				SteamSystem.currentSteams--;
			}
			else {
				LosingSteams.bulletsOnMoment = SteamSystem.currentSteams;
				LosingSteams.damaged = true;
				SteamSystem.currentSteams = 0;
				
			}
			Destroy(gameObject);
		}
	}
}
